### ISSUE TITLE
Image loading blocking UI

### ISSUE THREAD

**Issue by nselvapandian** [M2Mobi/Marky-Mark#78]
While rendering big size image, app got freezed because it's blocked the UI. It is possible to render images in background without blocking the UI.

**Comment by georgbachmann** [comment]
Just had the exact same problem. App freezes quite some time...
And in the mean-time. Is there a way to disable inline images? (Better not showing them than having the app freeze)

**Comment by Basca** [comment]
Hey nselvapandian and georgbachmann,

We are aware of rendering that can block UI for the inline images and are still looking for a good way to fix the issue. Feel free to look into the issue and share your thoughts on how we can improve it.

In the meantime, you can actually remove/replace the `InlineImageAttributedStringBlockBuilder` by creating your own subclass of `MarkDownConverterConfiguration` and passing that to the `MarkDownConverter`.
An example of how you can pass your own configuration can be found in `AdvancedMarkdownViewController`

In short:
```
        let configuration = <your custom subclass>
       let converter = MarkDownConverter(configuration: configuration)
```
